### ISSUE TITLE
Version RC-1 Cutoff generation

### ISSUE THREAD

**Issue by shredderslash** [TerraForged/tracker#10]
Ever since updating to RC-1 I've been getting world generation like this.
![2021-01-16_13 57 46](https://user-images.githubusercontent.com/8062717/104822851-d31cf000-5802-11eb-9e98-27ca94452d04.png)

![2021-01-16_13 48 03](https://user-images.githubusercontent.com/8062717/104822754-2e9aae00-5802-11eb-813e-8fe6862c9a44.png)

**Comment by Won-Ton** [comment]
Is this on a new world? otherwise provide the exact steps to reproduce please

**Comment by shredderslash** [comment]
it happened on an old world but both screenshots are from new worlds. the bottom one I had made some changes to the world gen settings so I made a new world without touching anything to see if it was a setting I had changed that was causing the issue the results of which is the world the top screenshot came from. All I did for the top one was make a new terraforged world, set it to creative mode, then wandered around till I found that half-mountain.
